src/client/src/views/SceneForm.js

```diff
@@ -25,34 +25,46 @@ module.exports = {
       }, [
         m("input.new-name-input[type=text]", {
           oninput: function (e) {
-            Scene.current.name = e.target.value
+            Scene.currentScene.name = e.target.value
           },
-          value: Scene.current.name ? Scene.current.name : `Scene ${Scene.current.id}`
+          value: Scene.currentScene.name ? Scene.currentScene.name : `Scene ${Scene.currentScene.id}`
```

Comment:
Make this `value: Scene.currentScene.name ? Scene.currentScene.name : ''` please